src/bridge/AdminRoomHandler.ts

```diff
@@ -213,6 +221,47 @@ export class AdminRoomHandler {
         return new MatrixAction("notice", "Room plumbed.");
     }
 
+    private async handleUnlink(args: string[], sender: string, userPermission: string | undefined) {
+        if (userPermission !== 'admin') {
```

Comment:
This would block anyone who isn't a bridge admin, but we'd like the logic to be `(isbridgeAdmin || isPriviledgedInRoom)`.

Reply:
:+1: I removed that requirement.